packages/connect/src/relay/connection.ts

```diff
@@ -1,18 +1,27 @@
-import { Multiaddr } from 'multiaddr'
```

Comment:
API changes + simplified passing of WebRTC instance using ConnectComponents and Getter function